### ISSUE TITLE
Crash when leaving a community and going to the portal

### ISSUE THREAD

**Issue by jrainville** [status-im/status-desktop#16437]
# Bug Report

## Description

[crash-leave.webm](https://github.com/user-attachments/assets/7a6ce0c0-5ef9-4138-8786-08e268cb7389)

Nothing interesting in logs:

```
INF 2024-10-01 19:15:45.623Z active mailserver changed                  topics="mailservers-service" tid=379962 file=service.nim:155 node=/dns4/store-02.gc-us-central1-a.status.prod.status.im/tcp/30303/p2p/16Uiu2HAmJnVR7ZzFaYvciPVafUXuYGLHPzSUigqAmeNw9nJUVGeM id=store-02.gc-us-central1-a.status.prod
ACTIVE MAILSERVER CHANGED: ref 0x7633c6e96bf0 --> [nodeAddress = 0x7634b5688240"/dns4/store-02.gc-us-central1-a.status.prod.status.im/tcp/30303/p2p/16Uiu2HAmJnVR7ZzFaYvciPVafUXuYGLHPzSUigqAmeNw9nJUVGeM",
nodeId = 0x7633c6cb6240"store-02.gc-us-central1-a.status.prod"]
INF 2024-10-01 19:15:45.623Z mailserver available                       topics="mailservers-service" tid=379962 file=service.nim:160
INF 2024-10-01 19:15:45.623Z history request started                    topics="mailservers-service" tid=379962 file=service.nim:170 numBatches=2
DBG 2024-10-01 19:15:45.749Z [threadpool task thread] initiating task   topics="task-threadpool" tid=379988 file=threadpool.nim:56 messageType=AsyncLoadCuratedCommunitiesTaskArg:ObjectType threadid=379988 task="{\"$type\":\"AsyncLoadCuratedCommunitiesTaskArg:ObjectType\",\"vptr\":101626889922752,\"slot\":\"onAsyncLoadCuratedCommunitiesDone\"}"
DBG 2024-10-01 19:15:45.750Z NewBE_callPrivateRPC                       topics="rpc" tid=379988 file=core.nim:27 rpc_method=wakuext_curatedCommunities
DBG 2024-10-01 19:15:45.751Z [threadpool task thread] initiating task   topics="task-threadpool" tid=379985 file=threadpool.nim:56 messageType=AsyncAddCentralizedMetricIfEnabledTaskArg:ObjectType threadid=379985 task="{\"$type\":\"AsyncAddCentralizedMetricIfEnabledTaskArg:ObjectType\",\"eventName\":\"navigation\",\"eventValueJson\":\"{\\\"viewId\\\":\\\"communitiesPortal\\\"}\",\"vptr\":101626889863488,\"slot\":\"onCentralizedMetricAddedIdEnabled\"}"
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [Makefile:824: run-linux] Segmentation fault (core dumped)
jonathan@jonathan-ThinkPad-P15-Gen-2i:~/dev/status-desktop2$ 
```

## Steps to reproduce

0. Join a community
1. Leave community
2. Go to the portal

#### Expected behavior

Portal opens

#### Actual behavior

Crash

### Additional Information

- Status desktop version: master `ee9091377f3e9b8dfc1a805c48e38101f456b9e4`
- Operating System: Ubuntu 22

**Comment by caybro** [comment]
@jrainville it crashes here:
![image](https://github.com/user-attachments/assets/e7af072f-7d5e-4bc0-88ef-3985750ec7ce)

**Comment by jrainville** [comment]
Clising as this has been fixed by @micieslak is `f8d33656e6e195cb41ab121100f7327a45f4d107`
